docs/setup-guide/scripts/README.md

```diff
@@ -0,0 +1,97 @@
+# Fornax Cluster Multi-Layer: Multi-Edge Node in the same level Setup using Bash Scripts
```

Comment:
please rename this file to more descriptive name, otherwise it'll be confused for the README of this entire folder

Reply:
rename to : scripts-> edge_cluster_setup_multiple_edgecore_nodes -> sh all file and readme.md